src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests.cs

```diff
@@ -35256,17 +35236,18 @@ static async Task<int[]> M2()
             verifier.VerifyDiagnostics();
             verifier.VerifyIL("C.<Main>d__0.System.Runtime.CompilerServices.IAsyncStateMachine.MoveNext()", """
                 {
-                  // Code size      544 (0x220)
+                  // Code size      498 (0x1f2)
                   .maxstack  4
                   .locals init (int V_0,
                                 System.Collections.Generic.List<int> V_1, //items
                                 int[] V_2,
                                 int[] V_3,
-                                System.Span<int> V_4,
-                                System.ReadOnlySpan<int> V_5,
-                                System.Runtime.CompilerServices.TaskAwaiter<int[]> V_6,
-                                System.Span<int> V_7,
-                                System.Exception V_8)
+                                int V_4,
```

Comment:
This seems like a nice little win. It looks like the local is no longer being lifted into a display class because now it is not used after the `await`.